guides/source/active_job_basics.md

```diff
@@ -300,13 +293,63 @@ end
 ProcessVideoJob.perform_later(Video.last)
 ```
 
+If you want more control on what queue a job will be run you can pass a `:queue`
+option to `set`:
+
+```ruby
+MyJob.set(queue: :another_queue).perform_later(record)
+```
+
 NOTE: Make sure your queuing backend "listens" on your queue name. For some
 backends you need to specify the queues to listen to.
 
 [`config.active_job.queue_name_delimiter`]: configuring.html#config-active-job-queue-name-delimiter
 [`config.active_job.queue_name_prefix`]: configuring.html#config-active-job-queue-name-prefix
 [`queue_as`]: https://api.rubyonrails.org/classes/ActiveJob/QueueName/ClassMethods.html#method-i-queue_as
 
+Queue Priority
+--------------
+
+Most adapters support queue priorities. You can schedule a job to run with a specific queue priority using [`queue_with_priority`][]:
```

Comment:
I think there are only 3 adapters that supports queue priorities at the job level. The others either don't support it or supporst it at the queue level https://github.com/rails/rails/blob/80bce4aad3b288a92745abb7c3d15e5bb8aa2c82/activejob/lib/active_job/queue_adapters.rb#L28-L30

We should probably add in this guide that the `queue_with_priority` feature only works for adapters that supports priority at the job level

Reply:
> We should probably add in this guide that the `queue_with_priority` feature only works for adapters that supports priority at the job level

Ah maybe I misunderstood how `queue_with_priority` works. I knew the implementation varied by adapter, but I didn't realise that it only worked with job level supporting adapters...

I'll update to clarify.

Reply:
> On a related note, I'm suprised that [good_job](https://github.com/bensheldon/good_job) isn't in that list. [solid_queue](https://github.com/basecamp/solid_queue) as well.

I think we stopped accepting adding adapters to the framework (i.e. the good_job adapter lives in its own gem). For solid queue, it looks like the doc will be updated when it becomes the default in Rails 8 https://github.com/rails/rails/pull/50791#issuecomment-1898868491